### PR TITLE
Changes as per the new Share strucuture schema

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#4553

*Description of changes:*
Changes in share structure

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/mixins/filing-template-mixin.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default class FilingTemplateMixin extends Vue {
     this.setOrgPersonList(filing.incorporationApplication.parties)
 
     // Set Share Structure
-    this.setShareClasses(filing.incorporationApplication.shareClasses)
+    this.setShareClasses(filing.incorporationApplication.shareStructure.shareClasses)
```
Comment (severinbeauvais) [inline src/mixins/filing-template-mixin.ts:134]: Should this check whether shareStructure is truthy first, as in
https://github.com/bcgov/bcrs-business-create-ui/blob/a1879ad75aace3a8b4c793aba79516f1c1a94774/src/mixins/filing-template-mixin.ts#L129
?

**Review by severinbeauvais — COMMENTED** [review]
Thanks for updating our new project 👍

File: src/mixins/filing-template-mixin.ts (hunk 1 above)
Comment (lekshmimallika-aot) [reply src/mixins/filing-template-mixin.ts:134]: Share structure will be truthy in our completed incorporation application. The reason why we have the check in other place is for the temp reg when we start the filing we will only have the nameRequest within it.. share structure wont be present will we save it / file it. It is not the case when it comes to edit UI.

However, there are some applications in prod that are completed and will have the old share structure. But we have an action item to correct the json of those completed filings in prod and should be done before this UI gets deployed to prod

**Review by cameron-freshworks — APPROVED** [review]
Excellent, thanks for this!

File: src/mixins/filing-template-mixin.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/mixins/filing-template-mixin.ts:134]: I think we should check anyway, please and thanks!

**Review by severinbeauvais — APPROVED** [review]
Thanks for the safety check!
Looks good.

**Pull request merged**